samples/RaspberryPiInstructions.md

```diff
@@ -0,0 +1,68 @@
+# .NET Core on Raspberry Pi
+
+Arm32 support for .NET Core is still being brought up but there has been enough progress for people to start playing with it. 
+It's not ready to be used as a developer environment yet but you can publish an application on a developer machine and run it on a Raspberry Pi. 
+These steps have been tested on both a Pi 2 and Pi 3 on Linux and Windows with success.
+
+##Creating an app:
+
+* [Install .NET Core](http://dotnet.github.io/getting-started/) into a supported developer configuration.
+
+* From the terminal/commandline create a folder named helloworld and go into it.
+
+* Run `dotnet new` then `dotnet restore` then `dotnet publish` to publish your helloworld app.
+
+* Run `dotnet ./bin/Debug/netcoreapp1.1/publish/helloworld.dll` on Linux/OSX or `dotnet .\bin\Debug\netcoreapp1.1\publish\helloworld.dll` and you should see it print `Hello World!`.
+
+* Under `./bin/Debug/netcoreapp1.1/publish` or `.\bin\Debug\netcoreapp1.1\publish` on Windows you need to edit the helloworld.runtimeconfig.json file to update the version to `1.2.0-beta-001206-00`.
+
+* This app is now ready to copy over to any machine that has the `1.2.0-beta-001206-00` version of .NET Core on it.
+
+
+##Getting the app to run on the Pi.
+
+###Linux (Ubuntu)
+
+* Install [Ubuntu 14.04 or 16.04](https://www.raspberrypi.org/downloads/) on your Pi.
```

Comment:
Debian would work if we made the zips for them. I haven't tried these bits on debian but they may work. I don't know all the reasons why we have seperate debian and ubuntu builds.